.travis.yml

```diff
@@ -148,6 +148,20 @@ jobs:
         INSTALL_AIRFLOW_VERSION="1.10.9"
       stage: test
       script: ./scripts/ci/ci_prepare_and_test_backport_packages.sh
+    - name: "Build production image Py3.6"
+      env: >-
+        PYTHON_MAJOR_MINOR_VERSION="3.6"
+      stage: test
+      script: ./scripts/ci/ci_build_production_image.sh
+      before_install:
+        - echo
+    - name: "Build production image Py3.7"
+      env: >-
+        PYTHON_MAJOR_MINOR_VERSION="3.7"
+      stage: test
+      script: ./scripts/ci/ci_build_production_image.sh
+      before_install:
+        - echo
```

Comment:
Should we build the image every time?

Reply:
Yes. The old rule is that the only way you can test if something works is to run it often. I believe in running everything we want to keep working via CI environment. This run will be rather fast usually (2-3 minutes) as it will be using cache once we push the image to dockerhub.